### DISCUSSION TITLE
blog/2023/01/08/bibdesk-to-zotero-pandoc/index

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#78] [Blog comments]
# blog/2023/01/08/bibdesk-to-zotero-pandoc/index

Tips, tricks, and rationale for converting from a single big BibTeX file to a Zotero database

https://www.andrewheiss.com/blog/2023/01/08/bibdesk-to-zotero-pandoc/index.html

**Comment by nathan-artist** [comment]
I submitted a pull request on GitHub with a a textual change to this post noting that BibDesk has a feature analogous to Zotero collections, which this blog post neglects to mention. The post currently states that BibDesk "uses only keywords" as if that were the only way to organize publications in BibDesk, which is not true. The pull request and its proposed changes can be seen here: https://github.com/andrewheiss/ath-quarto/pull/77
